tests/msim/action3.py

```diff
@@ -1,15 +1,12 @@
-import math
-
 def run(ecl_state, schedule, report_step, summary_state, actionx_callback):
 
-    wells = []
     if report_step == 1:
-        wells = ["P1"]
+        schedule.shut_well("P1", report_step)
     elif report_step == 2:
-        wells = ["P2"]
+        schedule.shut_well("P2", report_step)
     elif report_step == 3:
-        wells = ["P3"]
+        schedule.shut_well("P3", report_step)
     elif report_step == 4:
-        wells = ["P4"]
-
-    actionx_callback("CLOSEWELL", wells)
+        schedule.shut_well("P4", report_step)
```

Comment:
I think this looks. One comment/question:

Whenever I read such a line, I would be tempted to try
```
schedule.shut_well("P4", report_step + 1) # Maybe even use a value higher than the number of report steps.
```
What would happen?

Reply:
In case the argument exceeds the total number of report steps, an error is thrown.
If the argument does not exceed the total number of report steps, then the function call is scheduled for the given report step.